cmd/root.go

```diff
@@ -100,7 +100,10 @@ and other Red Hat SRE tools`,
 		o.ExecPostRunNonBlockingCmds()
 
 		if !viper.GetBool("--headless") {
-			o.Attach()
+			err = o.Attach()
+		}
+		if err != nil {
+			return err
```

Comment:
could we just `return o.Attach()` here? I assume the "attachment" is the last thing we want to do in a command?

Reply:
This is specifically a linter fix, but otherwise just as you say.  It's not really useful IRL.